src/critic_tests/conftest.py

```diff
@@ -1,9 +1,19 @@
 from moto import mock_aws
 import pytest
-
+import os
 
 @pytest.fixture(autouse=True)
 def mock_aws_all():
     """Make sure none of these tests are actually hitting AWS."""
     with mock_aws():
         yield
+
+
+@pytest.fixture(autouse=True)
+def aws_credentials():
```

Comment:
This brings up the question of how the lambda will get access to the resources it needs in production.

@rsyring is mu opinionated on how lambda tasks get access to AWS creds?